ehcache-impl/build.gradle

```diff
@@ -50,6 +50,7 @@ task slowTest(type: Test) {
 
 dependencies {
   api project(':ehcache-core')
+  implementation project(':clustered:ehcache-common')
```

Comment:
clustered modules should not be added as a dependency in ehcache impl

Reply:
Please look into possibility of extracting the files to a "shared" module which becomes a dependency for both ehcache-impl & clustered. That would also serve the intent of reusing the existing code.